### DISCUSSION TITLE
touch variant

### DISCUSSION THREAD

**Discussion by nendezkombet** [bjaan/roland-juno-g-display-replacement#2] [Q&A] [answered]
can i use touch variant lcd display altough does not use it

**Comment by bjaan** [comment, marked as answer]
Yes, you should be able to use the non-touch variant of the display, as is it just cheaper to purchase. You might have to remove the touch sensor as well. I recommend to use the "Module No Touch" version from the AliExpress link

**Comment by bjaan** [comment]
Closing.
